src/main/java/com/heatheryou/httpserver/handler/IHandler.java

```diff
@@ -0,0 +1,8 @@
+package com.heatheryou.httpserver.handler;
+
+import com.heatheryou.httpserver.Request;
+import com.heatheryou.httpserver.Response;
+
+public interface IHandler {
```

Comment:
Consider grouping interfaces with the classes that will use them, not the classes that implement them. It's also a good practice to try to find descriptive names for interfaces rather than just sticking on an `I` or `Impl` in front.

Reply:
Hmm.. Since the Router class is the one that uses the handler classes, do you mean I should group the Router and Handler interface together?

Reply:
Changed IHandler to RequestHandler in 59da68aa8183390c2a0fb1cfaf7c68edeea1d592

Reply:
>Since the Router class is the one that uses the handler classes, do you mean I should group the Router and Handler interface together

That's it!